drizzle-orm/src/pg-core/columns/postgis_extension/utils.ts

```diff
@@ -45,3 +45,64 @@ export function parseEWKB(hex: string): [number, number] {
 
 	throw new Error('Unsupported geometry type');
 }
+
+export function parseMultiLineStringEWKB(hex: string): [number, number][][] {
```

Comment:
Disclaimer: I did this function with AI, it may have mistakes. My PostgreSQL knowledge is not that deep 😅